src/lib/ngsi/EntityId.cpp

```diff
@@ -38,7 +38,7 @@
 *
 * EntityId::EntityId -
 */
-EntityId::EntityId() : keyName("entityId")
+EntityId::EntityId()
```

Comment:
Not actually related with this PR but I found that `keyName` was no longer used when modifying class fields, so I have take the opportunity to remove it.

NTC (just informative)